samples/server/petstore/python-aiohttp-srclayout/tox.ini

```diff
@@ -8,4 +8,4 @@ deps=-r{toxinidir}/requirements.txt
 	 {toxinidir}
 
 commands=
-   pytest --cov=src/openapi_server
+   pytest --cov=openapi_server
```

Comment:
Could maintainers advise me on the best change to apply in the mustache template to get this output ? The syntax is complex and I'm afraid to cut too much in what I don't know there: https://github.com/OpenAPITools/openapi-generator/blob/53289263d921e627cb05894c8477238dff48d62f/modules/openapi-generator/src/main/resources/python-aiohttp/tox.mustache#L11

Reply:
to remove `src`, what about removing `{{#lambda.forwardslash}}{{{pythonSrcRoot}}}{{/lambda.forwardslash}}` ? would that work?

Reply:
It generates the expected outcome in the sample, so if that's good to you that's good to me! Thank you for the suggestion.